docs/data/base/components/badge/badge-pt.md

```diff
@@ -89,7 +89,7 @@ Note that `componentsProps` slot names are written in lowercase (`root`) while `
 ## Hook
 
 ```jsx
-import { useBadge } from '@mui/base/BadgeUnstyled';
+import useBadge from '@mui/base/useBadge';
```

Comment:
Do not edit the localized (`-pt` and `-zh`) versions of the docs files. They are generated by Crowdin. Please revert changes to these files.